gtsam/hybrid/DCFactorGraph.h

```diff
@@ -39,6 +39,10 @@ class DCFactorGraph : public FactorGraph<DCFactor> {
       }
     }
 
+    // STL trick to remove duplicate keys
+    std::sort(result.begin(), result.end());
```

Comment:
Is it the correct thing to do? I think the order of the keys indicate the ordering used to eliminate, is it?

Reply:
Good point. I wasn't too happy about sorting either so let me see if I can use something else.

Reply:
Updated!